gradoop-accumulo/src/main/java/org/gradoop/common/storage/impl/accumulo/constants/AccumuloTables.java

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright © 2014 - 2018 Leipzig University (Database Research Group)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradoop.common.storage.impl.accumulo.constants;
+
+/**
+ * accumulo table store constants
+ */
+public class AccumuloTables {
```

Comment:
@s1ck do we have coding conventions on classes containing only static string constants? shoudln't  this be an ENUM or is this ok for gradoop?

Reply:
I don't think we have a convention for that. Personally, I don't have a favorite. We use those config classes at other places, too. Enums would require an additional constructor and toString implementation. Interfaces would work too. Whatever you prefer.